### DISCUSSION TITLE
Observability/OpenTelemetry

### DISCUSSION THREAD

**Discussion by giscus** [isno/theByteBook#224] [General]
# Observability/OpenTelemetry

构建大规模高可用的分布式系统

https://www.thebyte.com.cn/Observability/OpenTelemetry.html

**Comment by echo1937** [comment]
这里的章节有问题，直接就来到了9.4.3

**Comment by forrestIsRunning** [comment]
行未 “指标、链路追踪、指标....”，其中的指标重复

**Reply by isno** [reply]
谢谢，已修正。
